GiniVision/Classes/Core/CameraViewController.swift

```diff
@@ -122,7 +151,8 @@ public typealias CameraScreenFailureBlock = (_ error: GiniVisionError) -> Void
     fileprivate var detectedQRCodeDocument: GiniQRCodeDocument?
     fileprivate var currentQRCodePopup: QRCodeDetectedPopupView?
 
-    var didShowCamera: (() -> Void)?
+    public var didShowCamera: (() -> Void)?
+    public var didTapMultipageReviewButton: (() -> Void)?
```

Comment:
I'm not sure about these closures. You have four callbacks. Maybe it would be better to have a delegate?

Reply:
I don't like neither, actually they displease me a lot. But since we have an initializer with two closures (that I have never liked), it would suppose a breaking change for those who use the Component API.

Reply:
Does it make sense to compile a list of such things (items that we cannot change because they would be breaking changes, but we would like to). We could actually do them when we decide to release a major version.

Alternatively, we ca start deprecating things.